app/src/main/java/com/codesoom/assignment/MyHttpHandler.java

```diff
@@ -0,0 +1,151 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+
+import static com.codesoom.assignment.HttpStatusCode.*;
+
+public class MyHttpHandler implements HttpHandler {
+
+    JSONConverter jsonConverter = new JSONConverter();
+    TaskRepository taskRepository = new TaskRepository();
+    private Long idInPath;
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+
+        HttpRequest httpRequest = new HttpRequest(exchange);
+        String content = "Hello, world!";
+
+
+        if (!httpRequest.hasMethod().isEmpty()) {
+            switch (httpRequest.hasMethod()) {
+                case "GET":
+                    // /tasks만 입력했을 경우 (아이디로 검색하지 않고 전체 목록 얻을 때)
+                    if (httpRequest.hasPath().equals("/tasks")) {
+                        GETAllTaskList();
+                        content = JSONConverter.tasksToJSON(TaskRepository.getTaskStore());
+                        response(HTTP_OK, content, exchange);
+                        break;
+                    }
+                    // tasks/{id}를 입력했을 경우 (입력한 아이디에 해당하는 할일만 보여줌)
+                    Long id = getIdFromPath(httpRequest);
```

Comment:
`httpRequest`에서 `id`를 얻어온다는 것이 이해하기 어렵네요. 어떻게 하면 path 파라미터를 가져온다는 것을 의도를 드러낼 수 있을까요?

Reply:
```suggestion
                    // tasks/{id}를 입력했을 경우 (입력한 아이디에 해당하는 할일만 보여줌)
                    Long id = httpRequest.getId();
                    findOne(id);
                    content = jsonConverter.taskToJson(findOne(id));
                    response(HTTP_CREATED, content, exchange);
                    break;
```
위와 같이 수정해 보았습니다! 피드백 주신 의도에 맞게 수정한 것인지 확인 부탁 드리겠습니다 :)